sparse/unit_test/Test_Sparse_mdf.hpp

```diff
@@ -90,13 +92,90 @@ void run_test_mdf() {
     }
   }
   printf("}\n");
-
   EXPECT_TRUE(success)
       << "The permutation computed is different from the reference solution!";
 
+  // Check the factors L and U
   handle.sort_factors();
   crs_matrix_type U = handle.getU();
   crs_matrix_type L = handle.getL();
+  KokkosSparse::sort_crs_matrix<crs_matrix_type>(L);
```

Comment:
Didn't ``handle.sort_factors();`` already do this?

Reply:
Yep